ci/ray_ci/tester.py

```diff
@@ -288,6 +300,8 @@ def _get_test_targets(
         .strip()
         .split("\n")
     )
+    # remove additional trailing whitespace on windows OS
+    test_targets = {t.rstrip() for t in test_targets}
```

Comment:
how about changing line 300 to `split("\n")` and then `strip()`

the reason that you need this is because line break are 2 chars on windows. in fact, you can use `os.linesep` on the splitting: 

https://docs.python.org/3/library/os.html#os.linesep

Reply:
#verynice